File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (Java)/src/mypack/ThuongHieu.java
package mypack;

public class ThuongHieu {

	public String maThuongHieu;     // primary key
    public String tenThuongHieu;
    
	// constructor
    public ThuongHieu() { }
    public ThuongHieu(ThuongHieu thuongHieu)
    {
        this.maThuongHieu = thuongHieu.maThuongHieu;
        this.tenThuongHieu = thuongHieu.tenThuongHieu;
    }
    public ThuongHieu(String maThuongHieu, String tenThuongHieu)
    {
        this.maThuongHieu = maThuongHieu;
        this.tenThuongHieu = tenThuongHieu;
    }
    
    // getter - setter
    public String getMaThuongHieu() {
		return maThuongHieu;
	}
	public void setMaThuongHieu(String maThuongHieu) {
		this.maThuongHieu = maThuongHieu;
	}
	public String getTenThuongHieu() {
		return tenThuongHieu;
	}
	public void setTenThuongHieu(String tenThuongHieu) {
		this.tenThuongHieu = tenThuongHieu;
	}
}

File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/ThuongHieu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyShopQuanAo
{
    public class ThuongHieu
    {
        public string maThuongHieu { get; set; }     // primary key
        public string tenThuongHieu { get; set; }

        // constructor
        public ThuongHieu() { }
        public ThuongHieu(ThuongHieu thuongHieu)
        {
            this.maThuongHieu = thuongHieu.maThuongHieu;
            this.tenThuongHieu = thuongHieu.tenThuongHieu;
        }
        public ThuongHieu(string maThuongHieu, string tenThuongHieu)
        {
            this.maThuongHieu = maThuongHieu;
            this.tenThuongHieu = tenThuongHieu;
        }
    }
}
